components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -81,27 +84,54 @@ export default {
 	},
 };
 
+// ********* DOCS ONLY ********* //
```

Comment:
Just a nit-pick/opinion here- since we're using the `Default` story for chromatic testing (so it isn't really "docs-only"), maybe this comment should get moved beneath it.